modules/acquia_cms_headless/src/Service/StarterkitNextjsService.php

```diff
@@ -744,4 +742,18 @@ public function dashboardDestination(): array {
     ];
   }
 
+  /**
+   * A function that tells whether site is having nextjs headless.
+   *
+   * @return int
+   *   TRUE or FALSE.
+   */
+  public function isHeadlessConsumerData(): int {
```

Comment:
@apathak18 - Can we use a function name that describe it best?

Reply:
@chandan-singh7929 Fixed as per suggestion and amend the changes.